yb-voyager/src/migassessment/sizing.go

```diff
@@ -625,9 +625,19 @@ func shardingBasedOnTableSizeAndCount(sourceTableMetadata []SourceDBMetadata,
 		var numColocated int = 0
 		var cumulativeObjectCount int64 = 0
 
+		var shardedObjects []SourceDBMetadata
+		var cumulativeSizeSharded float64 = 0
+
 		for _, table := range sourceTableMetadata {
 			// Check and fetch indexes for the current table
 			indexesOfTable, indexesSizeSum, _, _ := checkAndFetchIndexes(table, sourceIndexMetadata)
+			// DB-12363: make tables having more than 5 indexes as sharded(irrespective of size or ops requirements)
+			if len(indexesOfTable) > 5 {
```

Comment:
Can we have this as a static variable?

Reply:
Done added static variable COLOCATED_MAX_INDEXES_THRESHOLD